packages/app/screens/swipe-list.tsx

```diff
@@ -55,12 +55,16 @@ export const SwipeListScreen = withColorScheme(() => {
 const FeedSwipeList = () => {
   const { useParam } = createParam<Query>();
   const { data } = useFeed();
-  const [initialScrollIndex] = useParam("initialScrollIndex");
+  const [initialScrollItemId] = useParam("initialScrollItemId");
   const { bottom: safeAreaBottom } = useSafeAreaInsets();
+  const initialScrollIndex = data?.findIndex(
+    (item) => item.nft_id == initialScrollItemId
+  );
+
```

Comment:
why is this duplicated?

Reply:
Already in line 92

Reply:
yeah, they are in different components. do you mean something else?

Reply:
Missed that, just saw the same file name. So this should also be handled as remarked before (useMemo and handle -1 cases)